makefiles/pseudomodules.inc.mk

```diff
@@ -57,7 +57,7 @@ PSEUDOMODULES += log_color
 PSEUDOMODULES += lora
 PSEUDOMODULES += mpu_stack_guard
 PSEUDOMODULES += nanocoap_%
-PSEUDOMODULES += netdev_default
+PSEUDOMODULES += netdev_%
```

Comment:
... this line are the _actual_ problem. They are just declaring everything starting with `netdev_` a submodule of `netdev`. However, `netdev_tap` is not. It's just a `netdev` simulator using the hosts TAP interface (where the name is basically coming from). I would prefer, to either name the submodules explicitly, or do it in a similar way that I already tried in https://github.com/RIOT-OS/RIOT/pull/10970.